app/aboot/fastboot-msm8974.c

```diff
@@ -0,0 +1,56 @@
+#include <reg.h>
```

Comment:
Would prefer having an explicit SPDX license header here, you can choose between `GPL-2.0-only` (most lk2nd-specific things), `BSD-3-Clause` (qcom's additions to LK) and `MIT` (original LK). We have it all covered :D

I know some source files are currently missing it, need to fix it in the file where it's missing.

Also, if you have copied substantial code as-is from somewhere you may want to add the original copyright texts and use the same license.

Reply:
Added GPL-2.0-only.

> Also, if you have copied substantial code as-is from somewhere you may want to add the original copyright texts and use the same license.

Definitely not substantial, the closest is the couple of lines doing the bit shifting but that's just the way how to get the required bits from the variable and very trivial.